src/components/GameView/CardListSortable.vue

```diff
@@ -37,7 +37,7 @@
 
 <script>
 import GameCard from '@/components/GameView/GameCard.vue';
-import { sortBy } from 'lodash';
+import { orderBy } from 'lodash'; // Changed import
```

Comment:
I think you are completely right to swap this import and use `orderBy` instead. Once the code is updated though, I don't think the comment on this line will be helpful, because no one will need to know that the import used to be something else. No big deal, I think we can just take it out
```suggestion
import { orderBy } from 'lodash';
```

As a suggestion for how to frame your thinking about communicating these things, I like to use code comments for notes that are helpful to someone reading the current state of the code, and then to use PR comments (you can add comments to lines of the PR just like this in your own PR) to explain the reasoning for my changes. So this could be a use case for leaving a PR comment to note that you're changing the function used to sort, since this is the one that allows for sorting on multiple values. 

PR comments are good for describing how and why code is being changed, whereas code comments are good for describing why the code is the way it is right now, irrespective of changes. Just food for thought

Reply:
Hi @itsalaidbacklife I have added all the necessary comments and changed the version in package.json as requested, thanks for giving me this opportunity , I hope the code is up-to-date and the issue "Fix Sorting Order Issue in Scrap Pile" #477, has been resolved successfully, if it is so please accept the merge request.